server/scsynth/SC_CoreAudio.cpp

```diff
@@ -1598,6 +1606,9 @@ bool SC_CoreAudioDriver::DriverStart() {
     if (mWorld->mVerbosity >= 1) {
         scprintf("->SC_CoreAudioDriver::DriverStart\n");
     }
+
+    auto appIOProcFunc = mSafetyClipThreshold < INFINITY ? appIOProc<true> : appIOProc<false>;
```

Comment:
i think you should also disable clipping if the threshold is negative. it would rhyme with the 'negative = off' behavior that pops up in other places in SC. and also, the clipping behavior will not work if the input is negative; if i'm following the definition of sc_clip2 right, the output will just sit at the negative value most of the time, which could lead to loud pops as well.

Reply:
You're right, I think I got confused with dBs vs amp. Actually I think I intended to implement this interface in dBs, such as in `scsynth -s -6` or `scsynth -s 0` and then convert to amp before assignment. It feels more consistent with server.volume... but now I'm not so sure about it anymore.. any comment? Would it be better in amp or db for you?

Reply:
i don't have a strong opinion either way tbh, but i think working with amp over db is slightly easier for new users to understand.

Reply:
I have a slight preference to keep this in amp, not in dB. 
I never used server.volume programmatically - it makes sense that it's in dB, but pretty much everything else is in amp, so to me it feels more like an outlier.